website/preprints/tasks.py

```diff
@@ -0,0 +1,129 @@
+import datetime
+import urlparse
+
+import requests
+
+from framework.celery_tasks import app as celery_app
+
+from website import settings
+
+
+@celery_app.task(ignore_results=True)
+def on_preprint_updated(preprint_id):
```

Comment:
I'm fairly certain that this would not be a suitable async task for the job. This should be ran when preprints are created or modified, but the only way to verify that it is would be to check SHARE.

Assuming that everything is working fine, this would take on average about 5 minutes to be surfaced in SHARE results (longer if SHARE is slammed, as this call is also handled async on the SHARE side), which isn't great turnaround for an automated test. Also, this would need to "publish" a preprint in order to be observable, and we try to keep bad data out of SHARE.